### PR TITLE
data/events: Add connection_limit

### PR DESCRIPTION
**Pull request by sirainen**
DOV-6893

### CODE REVIEW

File: data/events.js [hunk 1]
```diff
@@ -1203,14 +1203,14 @@ compiling it at delivery).`
 		},
 		inherit: 'pre_login_client',
 		fields: {
-			reason: `Short reason; see the short to long reason mapping in the table below.`,
+			reason: `Short reason; see the short reason to description mapping in the table above.`,
```
Comment (cmouse) [inline data/events.js:1206]: could we use [[link,table_link]] here?

Comment (sirainen) [reply data/events.js:1206]: no idea, @slusarz ?

Comment (cmouse) [reply data/events.js:1206]: you can just use [[link, name of link]]

Comment (sirainen) [reply data/events.js:1206]: But this is all generated stuff. I'd need to link to "Field list", which is also generated. I don't know if I can somehow place an anchor there.

Comment (sirainen) [reply data/events.js:1206]: I'm guessing it would require changing the javascript code to generate anchors to all of them based on some template name.

Comment (slusarz) [reply data/events.js:1206]: Commonmark allows HTML, so no need to over complicate.  Just use HTML anchor linking.

i.e.:
```
reason: `Short reason; see the short reason to description mapping in the <a href="#pre_login_client_error_codes">table above</a>.`,
```

and

```
<span id="pre_login_client_error_codes">Error Codes:</span>
```

**Review by cmouse — APPROVED** [review]

**Pull request merged**